frontend/yarn.lock

```diff
@@ -2018,7 +2018,7 @@ acorn@^6.0.1, acorn@^6.0.4, acorn@^6.2.1:
   resolved "https://registry.yarnpkg.com/acorn/-/acorn-6.4.1.tgz#531e58ba3f51b9dacb9a6646ca4debf5b14ca474"
   integrity sha512-ZVA9k326Nwrj3Cj9jlh3wGFutC2ZornPNARZwsNYqQYgN0EsV2d53w5RN/co65Ohn4sUAUtb1rSUAOD6XN9idA==
 
-acorn@^7.1.1:
+acorn@^7.1.1, acorn@^7.2.0:
```

Comment:
Looks like a lot of dependencies were added to this file? Are these all transitive deps of axios?

Reply:
They're not all transitive deps of axios, but running the `yarn` command updates outdated deps/plugins as well. I just updated the PR description to include these changes as well